python/adelphi/README.md

```diff
@@ -0,0 +1,107 @@
+# Adelphi tooling
+
+## Commands
+
+### export-cql
+This command extracts schemas for the specified keyspaces from a Cassandra instance and then displays the CQL commands necesary to generate them to standard out.  You can optionally specify an output directory, in which case the CQL commands are written to files within that directory, one file for each keyspace.
+
+The following will display the schemas for the keyspaces "foo" and "bar" on standard out: 
+
+    adelphi --keyspaces foo,bar export-cql
+
+If you wish to store the schemas in a directory "baz" you could use the following instead:
+
+    adelphi --keyspaces foo,bar --output-dir=baz export-cql
```

Comment:
`keyspaces` has no `=` and output-dir` has it, I think that could be confusing. I tend do prefer having the equals sign, but I don't care much either way as long we're consistent.

Reply:
Good catch @grighetto , will fix in the next push

Reply:
This should be fixed now; I added an explicit "=" for all examples in the docs